### PR TITLE
Fix eslint errors

### LINKED ISSUE #70: Fix ESLint errors
**What you were expecting:**

- No red lines in the code.

**What happened instead:**

- Currently, the code shows the red lines at the bottom of itself.

**Steps to reproduce:**

- Fix `.eslintrc.json` and `.prettierrc` and `package.json`.

**Related code:**

* A link to a GitHub repo with the minimal codebase to reproduce the issue

```
insert short code snippets here
```

**Other information:**

**Environment** 

* 0Auth version:
* Last version that did not exhibit the issue (if applicable):
* Node version:
* Browser:
* Stack trace (in case of a JS error):

### PR DESCRIPTION
**Pull request by anthonyminyungi**
# Summary 

- It closes #70 

## How to Review? 

- Please refer to my self-review submitted as the review.

## Images 
- After changes in this PR.
스크린샷 2020-10-04 오후 8 44 23

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -26,12 +26,12 @@
...
-    "eslint-config-airbnb-base": "^14.2.0",
-    "eslint-config-prettier": "^6.11.0",
-    "eslint-plugin-import": "^2.22.0",
+    "eslint-config-prettier": "^6.12.0",
+    "eslint-plugin-prettier": "^3.1.4",
     "eslint-plugin-typescript": "^0.14.0",
     "mocha": "^8.1.1",
     "nyc": "^15.1.0",
+    "prettier": "^2.1.2",
     "tsconfig-paths": "^3.9.0",
```
Comment (anthonyminyungi) [inline package.json:35]: Deleted unnecessary dependencies and added prettier related dependencies.

File: .eslintrc.json [hunk 2]
```diff
@@ -7,11 +7,9 @@
...
   },
   "extends": [
-    "eslint:recommended",
-    "plugin:@typescript-eslint/eslint-recommended",
     "plugin:@typescript-eslint/recommended",
-    "plugin:@typescript-eslint/recommended-requiring-type-checking",
-    "airbnb-base"
+    "prettier/@typescript-eslint",
+    "plugin:prettier/recommended"
   ],
```
Comment (anthonyminyungi) [inline .eslintrc.json:13]: Deleted lines that have duplicated effects and added prettier related extensions.

File: packages/client/src/index.ts [hunk 3]
```diff
@@ -1,8 +1,18 @@
 import {
-  hashProperty, Property, PropertyType, Signature,
+  hashProperty,
+  Property,
+  PropertyType,
+  Signature,
 } from '@0auth/message';
 import {
```
Comment (anthonyminyungi) [inline packages/client/src/index.ts:7]: From this file, the changes below are because of the formatting.

**Review by anthonyminyungi — COMMENTED** [review]
Self-review.

**Review by HyeockJinKim — APPROVED** [review]
Cool.
I'll add lint CI.

**Pull request merged**